app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { fetchTopics } = require("./controllers/controller");
+const { handle500errors } = require("./controllers/errorHandlers");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
I mean you dont need this yes as youre not doing anything that needs to process the request from the user but its fine :)

Reply:
I know, I added it so I don't forget when I need it. I spent an hour last time debugging last time, didn't want to repeat that 😅